### ISSUE TITLE
[Bug] Bug for the command whitelist

### ISSUE THREAD

**Issue by Lennord** [BDXGaming/essentialsBungee#3]
Hello, I found several bugs to fix.

Bug 1:

When I do the command "/gwhitelist" and "/globalwhitelist", it displays nothing (normally it should put the correct display of the command but it does not work)

No Error in console !

Bug 2:

When I do the "/gwhitelist" or "/globalwhitelist" command, it shows me an error when I type 1 or 2 characters:

Example:
/gwhitelist add s
/gwhitelist add ss

/gwhitelist remove s
/gwhitelist remove ss

Error:
```
[WARNING] Error in dispatching command
java.lang.NullPointerException: Cannot invoke "String.length()" because "s" is null
    at java.base/java.io.StringReader.<init>(StringReader.java:51)
    at org.json.simple.parser.JSONParser.parse(JSONParser.java:79)
    at org.json.simple.parser.JSONParser.parse(JSONParser.java:75)
    at org.shanerx.mojang.Mojang.getJSONObject(Mojang.java:374)
    at org.shanerx.mojang.Mojang.getUUIDOfUsername(Mojang.java:110)
    at me.bdx.essentialsbungee.Utils.MojangPlayerHelper.getUniqueId(MojangPlayerHelper.java:31)
    at me.bdx.essentialsbungee.managers.WhitelistManager.removeWhitelistedUser(WhitelistManager.java:174)
    at me.bdx.essentialsbungee.commands.WhitelistCommand.execute(WhitelistCommand.java:54)
    at net.md_5.bungee.api.plugin.PluginManager.dispatchCommand(PluginManager.java:207)
    at net.md_5.bungee.api.plugin.PluginManager.dispatchCommand(PluginManager.java:157)
    at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:74)
    at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)
```

Have a good day !

**Comment by BDXGaming** [comment]
Both bugs have been resolved in 0.2.2 release.

**Comment by Lennord** [comment]
> Both bugs have been resolved in 0.2.2 release.

Okay thank you so much !
